.travis.yml

```diff
@@ -3,22 +3,27 @@ python:
   - 2.7
   - 3.4
   - 3.5
+  - 3.6
   - "nightly"
 matrix:
   allow_failures:
-    - python: 3.5
     - python: "nightly"
-sudo: false
 notifications:
   email:
     sos-devel@redhat.com
   irc:
     channels:
       - "us.freenode.net#sosreport"
     on_success: change
+dist: trusty
+sudo: true
```

Comment:
I believe so - It definitely gets us closer.  The reason I did the change to a VM was so we could use sudo.